docs/userguide.md

```diff
@@ -77,6 +78,25 @@ kubectl get all --all-namespaces
 Once all pods are running you can move on to deploy the cluster registry and
 federation-v2 control plane.
 
+#### Google Kubernetes Engine (GKE)
+
+[GKE](https://cloud.google.com/kubernetes-engine/) can be used for creating clusters instead of minikube. Since
+Federation v2 requires Kubernetes v1.11 or greater, you must create the clusters using the
+[Alpha Clusters](https://cloud.google.com/kubernetes-engine/docs/concepts/alpha-clusters) feature and manually specify
```

Comment:
These instructions will quickly go stale once GKE supports k8s >= v1.11. It would be good to add a comment about this.